packages/eslint-plugin/src/rules/strict-boolean-expressions.ts

```diff
@@ -715,11 +715,16 @@ export default util.createRule<Options, MessageId>({
             context.report({
               node,
               messageId: 'conditionErrorNullableObject',
-              fix: util.getWrappingFixer({
-                sourceCode,
-                node,
-                wrap: code => `${code} != null`,
-              }),
+              suggest: [
+                {
+                  messageId: 'conditionErrorNullableObject',
+                  fix: util.getWrappingFixer({
+                    sourceCode,
+                    node,
+                    wrap: code => `${code} != null`,
+                  }),
+                },
+              ],
```

Comment:
[Testing] And now that this is a `suggest` instead of a `fix`, we'll need unit testing for it. Could you add them in please?

`suggestions: [` is a good search term for other rules that give suggestions.